tidb-cloud/explore-data-with-chat2query.md

```diff
@@ -0,0 +1,110 @@
+
+## Limitation
+
+SQL queries generated by AI is not 100% accurate and might still need your further tweak.
+
+## Access Chat2Query
+
+1. Go to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
+
+    > **Tip:**
+    >
+    > If you have multiple projects, you can switch to the target project in the left navigation pane of the **Clusters** page.
+
+2. Click your cluster name, and then click **Chat2Query** in the left navigation pane.
+
+## Enable or disable AI to generate SQL queries
+
+PingCAP takes the privacy and security of users' data as top priority. The AI capacity of Chat2Query only needs to access database schemas to generate SQL queries, not your data itself. For more information, see [Chat2Query Privacy FAQ](https://www.pingcap.com/privacy-policy/privacy-chat2query).
+
+For the first-time you access Chat2Query, you will get a dialog about whether to allow PingCAP and OpenAI to use your code snippets to research and improve the services.
+
+- To enable AI to generate SQL queries, select the checkbox and click **Save and Get Started**.
+- To disable the AI to generate SQL queries, close this dialog directly.
+
+After the first-time access, you can still change the AI setting as follows:
+
+- To enable AI, click **Enable AI power for data exploration** in the upper-right corner of Chat2Query.
+- To disable AI, click <MDSvgIcon name="icon-top-account-settings" /> **Account** in the upper-right corner of the [TiDB Cloud console](https://tidbcloud.com/), click **Account Settings**, click the **Privacy** tab, and then disable the **AI-powered Data Exploration** option.
+
+## Write and run SQL queries
+
+In Chat2Query, you can write and run SQL queries using the pre-built sample dataset or your own dataset.
+
+1. Write SQL queries.
+
+    - If AI is enabled, simply type `--` and your instructions to let AI generate SQL queries automatically or write SQL queries manually.
+
+        For a SQL query generated by AI, you can accept it by pressing `Tab` and then further edit it if needed, or reject it by pressing `Esc`.
+
+    - If AI is disabled, write SQL queries manually.
+
+2. Run SQL queries.
+
+    <SimpleTab>
+    <div label="Mac">
+
+    For macOS:
+
+    - If you have only one query in the editor, to run it, press **Command + Enter** or click <svg width="1rem" height="1rem" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M6.70001 20.7756C6.01949 20.3926 6.00029 19.5259 6.00034 19.0422L6.00034 12.1205L6 5.33028C6 4.75247 6.00052 3.92317 6.38613 3.44138C6.83044 2.88625 7.62614 2.98501 7.95335 3.05489C8.05144 3.07584 8.14194 3.12086 8.22438 3.17798L19.2865 10.8426C19.2955 10.8489 19.304 10.8549 19.3126 10.8617C19.4069 10.9362 20 11.4314 20 12.1205C20 12.7913 19.438 13.2784 19.3212 13.3725C19.307 13.3839 19.2983 13.3902 19.2831 13.4002C18.8096 13.7133 8.57995 20.4771 8.10002 20.7756C7.60871 21.0812 7.22013 21.0683 6.70001 20.7756Z" fill="currentColor"></path></svg>**Run**.
```

Comment:
On macOS, I can use either **Control + Enter** or **Command + Enter** to run SQL.

Reply:
According to PM, this is to provide the same instructions as that on the UI.